src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -111,6 +111,13 @@ public TimeSpan QuickPulseInitializationDelay
         /// </summary>
         public DependencyTrackingOptions DependencyTrackingOptions { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value that filters logs before they are sent to Live Metrics. False by default.
+        /// When false, all logs are sent to Live Metrics, regardless of any log filter configuration. When true,
+        /// the logs are filtered before they are sent. Setting to true may yield better performance.
```

Comment:
On second thought, let's remove the last sentence about performance b/c we've not actually tested it. I don't want us to promise something that we haven't tested.